react-accordion-component/src/accordion.jsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+
+class Accordion extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      currenButtonDisplayed: ''
+    };
+    this.handleHTMLButton = this.handleHTMLButton.bind(this);
+    this.handleCSSButton = this.handleCSSButton.bind(this);
+    this.handleJSButton = this.handleJSButton.bind(this);
+  }
+
+  handleHTMLButton(event) {
+    if (event.target.className === 'active') {
+      event.target.className = '';
+      this.setState({ currenButtonDisplayed: '' });
+    } else {
+      event.target.className = 'active';
+      this.setState({ currenButtonDisplayed: 'HML' });
+    }
+  }
+
+  handleCSSButton(event) {
+    if (event.target.className === 'active') {
+      event.target.className = '';
+      this.setState({ currenButtonDisplayed: '' });
+    } else {
+      event.target.className = 'active';
+      this.setState({ currenButtonDisplayed: 'CSS' });
+    }
+  }
+
+  handleJSButton(event) {
+    if (event.target.className === 'active') {
+      event.target.className = '';
+      this.setState({ currenButtonDisplayed: '' });
+    } else {
+      event.target.className = 'active';
+      this.setState({ currenButtonDisplayed: 'JS' });
+    }
+  }
+
+  handleHypertextMarkupLanguageText() {
+    return this.state.currenButtonDisplayed === 'HML'
+      ? ''
+      : 'hidden';
+  }
+
+  handleCascadingStyleSheetsText() {
+    return this.state.currenButtonDisplayed === 'CSS'
+      ? ''
+      : 'hidden';
```

Comment:
The data being passed in should be able to be any topics passed in via props. This is hard coded for very specific data.